### PR TITLE
Prevent empty spaces from being counted as characters

### PR DESCRIPTION
**Pull request by gracelungu**
#### What does this PR do?
This PR refactors the `textarea` implementation to only count single spaces as characters

#### Description of Task to be completed?
~ Add a method that replaces multiple spaces by a single space
~ Use the created method to count the text input

#### How should this be manually tested?
Fetch and clone the branch `git fetch origin bg-fix-multiple-spaces-error && git checkout bg-fix-multiple-spaces-error`
Start the server and input multiple spaces in the post `textarea`

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Comment by gracelungu** [comment]
[Fix the post editor from posting empty text once multiple spaces are pressed](https://trello.com/c/aMnl5OVa/42-fix-the-post-editor-from-posting-empty-text-once-multiple-spaces-are-pressed)

File: src/components/PostTextArea/PostTextArea.jsx [hunk 1]
```diff
@@ -73,15 +73,20 @@ export class PostTextArea extends Component {
     }
   };
 
+  trimedTextLength = (text) => String(text).replace(/\s\s+/g, ' ').length;
+
   _onChange = (e) => {
     const textareaLineHeight = 24;
+    const text = e.target.value;
```
Comment (oesukam) [inline src/components/PostTextArea/PostTextArea.jsx:80]: Can't you destruct?

File: src/components/PostTextArea/PostTextArea.jsx [hunk 2]
```diff
@@ -73,15 +73,20 @@ export class PostTextArea extends Component {
...
+    const textLength = this.trimedTextLength(text);
+
     const {
       minRows, maxRows, minChar, maxChar, onChange,
     } = this.props;
     if (maxChar && maxChar <= e.target.value.length - 1) return;
```
Comment (oesukam) [inline src/components/PostTextArea/PostTextArea.jsx:86]: Why not use `text` from line 80 instead of accessing it from `e.target`

**Review by oesukam — CHANGES_REQUESTED** [review]
@gracelungu  Why not count in between empty spaces as characters?

**Pull request merged**